tictactoe.py

```diff
@@ -63,7 +25,7 @@ def play_a_turn(playername, board, first_player = True):
         try:
             player_input = input("Pick a number between 1 to 9 (inclusive).")
             if int(player_input) in range(1,10):
-                break
+                break #why break here? 
```

Comment:
<img width="596" alt="Screen Shot 2021-06-24 at 10 03 04 PM" src="https://user-images.githubusercontent.com/9858615/123372353-42c4a300-d538-11eb-8811-e8964d2466ca.png">
getting an error with number input

Reply:
I tried the same inputs (for names and first move of 2) but didn't return any error.